src/content/getting-started/observe-your-clusters-and-apps/index.md

```diff
@@ -69,47 +69,28 @@ Along with service monitor metrics, you can check metrics for DNS, Ingress, Flux
-  labels:
-    app.giantswarm.io/kind: dashboard
-  name: my-grafana-dashboard
-  namespace: my-namespace
-```
+Grafana data is stored in a PostGreSQL cluster with regular backups so your dashboard is persisted even if you upgrade it later on or if the system restarts. For more information on dashboard creation, you can check our [dashboard creation tutorial]({{< relref "/tutorials/observability/data-exploration/creating-custom-dashboards" >}}).
 
-It's important to have an annotation with the key `observability.giantswarm.io/organization` pointing to the organization where the dashboard will be stored in Grafana. Also notice the label `app.giantswarm.io/kind` has to be set to `dashboard` in order to be recognized by the platform.
+In our example, you can find the dashboard JSON [here](./dashboard.json). Download the dashboard content and import it into the Grafana UI:
 
-The easiest way to build a dashboard is using the Grafana UI. There, you can create panels with the desired visualizations and then export them by selecting `Share > Export`` in the dashboard context menu or by accessing the JSON Model in the dashboard settings.
+- Open the `Dashboards` view in the `Grafana` UI, click on the `New` button on the top right part of the view and then on the `import`one:
 
-__Warning__: The dashboards served in the platform are all created from a GitOps repository or created by users through `ConfigMap` resources; there is no persistent storage for `Grafana`, which means dashboards aren't persisted if the system restarts or are upgraded.
+![dashboard-import-new](dashboard-import.png)
 
-In our example, you can find the dashboard JSON [here](./dashboard.json). Download the dashboard content and use `jq` to minify it:
+- Upload the dashboard JSON file and click on `Load`:
 
-```sh
-jq -c . dashboard.json
-```
-
-Then replace the `__DASHBOARD_JSON__` placeholder in the `ConfigMap` with the minified JSON content. After that, apply the `ConfigMap` to the platform API:
+![dashboard-import-panel](dashboard-import-json.png)
 
-```sh
-kubectl apply -f config-map.yaml
-```
-
-After a few seconds, you can open the `Dashboards` view in the `Grafana` UI and, switch to the organization you selected, and find your custom dashboard.
+This will create the dashboard in the organization you are currently logged in:
 
 ![custom-dashboard](custom-dashboard.png)
 
 As you can see in the image above, the dashboard shows some information about the `hello-world` application. You see the number of errors, the number of success requests, and the number of requests by code. Feel free to poke around and create your own panels.
 
+Please note that the recommended way for creating dashboard is to use the `GitOps` approach where you store your dashboards' JSON files into a git repository. For more information, check the [related doc]({{< relref "/tutorials/observability/data-exploration/creating-custom-dashboards/#gitops" >}}).
```

Comment:
I would rather have this add the beginning of this section offering the possibility to the user to decide whether to continue with the ClickOps approach described here or use GitOps, I also don't think this is a recommended way anymore since we support both, it's just an alternative at this point.

Reply:
I'll move it up then.

> I also don't think this is a recommended way anymore since we support both, it's just an alternative at this point

We still suggest users to use the GitOps approach as it's more resilient and also allows you to define a cluster once for all the user's installations (if the user does have several ones)